src/librustc_target/spec/thumbv4t_nintendo_gba.rs

```diff
@@ -62,9 +62,6 @@ pub fn target() -> TargetResult {
             // ABIs to not use
             unsupported_abis: super::arm_base::unsupported_abis(),
 
-            // The minimum alignment for global symbols.
-            min_global_align: Some(4),
-
```

Comment:
You can drop dbfe8fc61c9420aca1679249749df6b9cd1c00df, that was my mistake. Adding `min_global_align` should be fine.

Reply:
Well once I was thinking about it, I realized that is **is** kinda weird and unexpected. It's also not required for the target to be correct, it just seemed neat for a moment when i was going through the huge TargetOptions struct.

I think that I really will leave it out, since you can align a value to 4 in user code using wrapper structs. That way, a particular value having a non-standard alignment for its type would be apparent within the code, instead of seemingly just happening at random.

Reply:
Okay, also seems fine.